packages/lit-dev-content/site/tutorials/content/intro-to-lit/05.md

```diff
@@ -38,21 +38,32 @@ You can use standard JavaScript in Lit expressions to create conditional or repe
     }
 
     addToDo() {
-      this.listItems.push({text: this.input.value, completed: false});
+      this.listItems = [...this.listItems,
+          {text: this.input.value, completed: false}];
       this.input.value = '';
-      this.requestUpdate();
     }
     ```
 
     {% endswitchable-sample %}
 
-    As the name suggests, `requestUpdate()` triggers the component to update. Setting a reactive property calls this automatically. Since you're not setting `listItems` here, but only mutating the array, you need to call `requestUpdate()` yourself.
-
-    The `@query` decorator is a handy way of getting a reference to a node in your component's internal DOM. It's basically equivalent to calling code like this in your constructor:
+    The `@query` decorator (used in the TypeScript version of the code) is a handy way of getting a reference to a node in your component's internal DOM. It's  equivalent to this code in the JavaScript version:
 
     ```js
-    this.input = this.shadowRoot.querySelector('#newitem');
+    get input() {
+      return this.renderRoot?.querySelector('#newitem') ?? null;
```

Comment:
Why change this to use `renderRoot`?

Reply:
I'm just adding the code that our TS->JS translator generates so this code matches the solved code.